package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "chat",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "engines": {
+    "node": ">12.0.0"
+  },
+  "scripts": {
+    "start": "PORT=3000 parcel src/pages/auth/auth.hbs --no-cache",
+    "prebuild": "engine-version",
+    "build": "parcel build src/pages/auth/auth.hbs  --no-minify --no-cache --public-url ./",
```

Лучше использовать "wildcard", чтобы на всякий случай не допустить варианта, когда сборщик не сможет сам распознать все зависимости между страницами и не получилось так, что в dist попадут не все страницы
```suggestion
    "start": "PORT=3000 parcel src/pages/**/*.hbs --no-cache",
    "prebuild": "engine-version",
    "build": "parcel build src/pages/**/*.hbs  --no-minify --no-cache --public-url ./",
```